docs/resources/service_user.md

```diff
@@ -10,6 +10,8 @@ description: |-
 
 The Service User resource allows the creation and management of Aiven Service Users.
 
+~> **Note:** This resource is deprecated. Please use service-specific resources instead of this one, for example: aiven_kafka_user, aiven_pg_user etc.
```

Comment:
isn't this is supposed to be autogenerated?

Reply:
Yes indeed we have a resource deprecation message for `aiven_service_user` but it is not added to the generated docs.

Reply:
I think it has something to do with the docs gen tool